webpack.config.js

```diff
@@ -7,10 +7,14 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const CompressionPlugin = require("compression-webpack-plugin");
 const ESLintPlugin = require('eslint-webpack-plugin');
 const CockpitPoPlugin = require("./src/lib/cockpit-po-plugin");
+const CockpitRsyncPlugin = require("./src/lib/cockpit-rsync-plugin");
 
 /* A standard nodejs and webpack pattern */
 const production = process.env.NODE_ENV === 'production';
 
+// Obtain package name from package.json
+const packageJson = JSON.parse(require('fs').readFileSync('package.json'));
```

Comment:
For consistency, please move the require() back to the top. This was fine for an one-liner in a github workflow, but not for a full program. We may need to use it in another place in the future, and don't want to import it more than once.